laurent22/joplin
# Chore: Remove `.gitignore`d file from version control

# Summary

`packages/app-desktop/gui/note-viewer/lib.js` [is auto-generated](https://github.com/laurent22/joplin/blob/01ec640bdb88cf07167bfd43f3dbd202381153f5/packages/app-desktop/tools/compileScripts.js#L12) from several files in `packages/lib`. It's also in [`packages/app-desktop/.gitignore`](https://github.com/laurent22/joplin/blob/01ec640bdb88cf07167bfd43f3dbd202381153f5/packages/app-desktop/.gitignore#L9), though was still tracked by version control.

https://github.com/laurent22/joplin/commit/01ec640bdb88cf07167bfd43f3dbd202381153f5 made a change to one of the files in `packages/lib/` that generates this file, causing `note-viewer/lib.js` to change on build.

This pull request removes `note-viewer/lib.js` from version control.

# Testing

1. Run `yarn build; yarn start` in `packages/app-desktop`
2. Verify that the note viewer can still be used to view notes.
3. Verify that `packages/app-desktop/gui/note-viewer/lib.js` has been created.

This has been successfully tested on Ubuntu 23.10.
